DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -218,7 +218,12 @@
   <ioread sourceClass = "pat::Jet" version="[1-11]" targetClass="pat::Jet" source="int partonFlavour_" target="jetFlavourInfo_">
     <![CDATA[jetFlavourInfo_ = reco::JetFlavourInfo(0,onfile.partonFlavour_);]]>
   </ioread>
-  <class name="pat::MET"  ClassVersion="14">
+  <class name="pat::MET"  ClassVersion="19">
+    <version ClassVersion="19" checksum="428901429"/>
+    <field name="corMap_" transient="true"/>
```

Comment:
Just go to version 15 here with the latest checksum, no need for intermediate versions